midterm-2023-11-07/code/5263-angela-kostadinova/script.ts

```diff
@@ -0,0 +1,106 @@
+interface Movie {
+
+id: number;
+title: string;
+director: string;
+year: number;
+genre: string;
+plot: string;
+cast: string[];
+oscars: number;
+}
+async function LoadingTheData() {
```

Comment:
General comment: the convention for JavaScript / TypeScript is to use lowerCamelCase for functions.